components/Account/Access/Campaigns/index.js

```diff
@@ -0,0 +1,75 @@
+import { compose, graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import Campaign from './Campaign'
+
+import query from '../../belongingsQuery'
+
+const Campaigns = ({ accessCampaigns, grantAccess, revokeAccess, state }) => {
```

Comment:
is `state` prop used? A bit unfortunate naming as it is then spread as props onto `<Campaign />`

Reply:
It is not used.